stories/core_components/GeoPushpinChart.tsx

```diff
@@ -135,8 +137,8 @@ storiesOf("Core components/GeoPushpinChart", module)
                 projectId: "storybook",
                 location: ATTRIBUTE_LOCATION_GEOCHART,
                 size: MEASURE_SIZE_GEOCHART,
-                color: MEASURE_COLOR_GEOCHART,
-                segmentBy: ATTRIBUTE_SEGMENT_GEOCHART,
+                color: MEASURE_COLOR_GEOCHART_ALIAS,
+                segmentBy: ATTRIBUTE_SEGMENT_GEOCHART_ALIAS,
```

Comment:
We should have new case named `with attrbute/measure alias in tooltip` instead of modifying current case without describing anything.
Name of storybook test or any test should tell Devs correct and concise context as possible.

Reply:
fixed